### ISSUE TITLE
phpcs:ignore breaks MultiLineArrayComma if used on last element

### ISSUE THREAD

**Issue by blastoise365** [squizlabs/PHP_CodeSniffer#2672]
This code will work fine:

```
$array = [
    'aaaaa',
    'bbbbb',
];
```

But this one will report MultiLineArrayComma problem

```
$array = [
    'aaaaa',
    'bbbbb', // phpcs:ignore
];
```

My example why I'm using ignore on last element: (I'm getting some results with guzzle and keys in response are not properly named so I'm ignoring camel case on those)

```
$bot->userStorage()->save([
    'x_task_option_'.$i.'_id' => $taskData->Results[$i]->id, // phpcs:ignore Zend.NamingConventions.ValidVariableName.NotCamelCaps
    'x_task_option_'.$i.'_name' => $taskData->Results[$i]->name, // phpcs:ignore Zend.NamingConventions.ValidVariableName.NotCamelCaps
]);
```

**Comment by gsherwood** [comment]
I'm not sure what sniff is reporting an error for you as I don't see that error code anywhere in the PHPCS source code, and none of the included array sniffs are reporting errors on the same code you've provided.

Can you run PHPCS with the `-s` command line argument and see what specific sniff code is being reported for the error please.

**Comment by blastoise365** [comment]
```
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------
  82 | ERROR | [x] Add a comma after each item in a multi-line
     |       |     array
     |       |     (Symfony.Arrays.MultiLineArrayComma.Invalid)
 170 | ERROR | [x] Add a comma after each item in a multi-line
     |       |     array
     |       |     (Symfony.Arrays.MultiLineArrayComma.Invalid)
 265 | ERROR | [x] Add a comma after each item in a multi-line
     |       |     array
     |       |     (Symfony.Arrays.MultiLineArrayComma.Invalid)

```

But it says that it can fix it automatically. I will try if that works.

**Comment by gsherwood** [comment]
That's a sniff from the Symfony custom coding standard and not a sniff that is included with PHPCS itself.

You might be better off speaking to whoever maintains that project and seeing why the sniff is not able to cope with comments after array commas.

**Comment by blastoise365** [comment]
Funny thing is that I've started with writing issue in their repo and I don't know why I have decided to discard it and open it here first :) Sorry, I will close it here.

**Comment by gsherwood** [comment]
No problem. If they think it is a core PHPCS issue and it can be replicated easily enough, please reopen.
